docs/content/SegmentedControl.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: SegmentedControl
+status: Draft
+description: Used to select an option from a short list and immediately apply the selection
```

Comment:
Could we write this in the same style as other descriptions in [primer/component-metadata](https://github.com/primer/component-metadata#metadata) so it can eventually be moved there?

```suggestion
description: Use a segmented control to let users select an option from a short list and immediately apply the selection
```